src/main/resources/add-ons-to-index.json

```diff
@@ -3448,6 +3568,9 @@
       "uid": "org.openmrs.module.reference-demo-data-module",
       "type": "OMOD",
       "name": "Reference Demo Data Module",
+	  "tags": [
+        "data-management", "demo", "reference-application"
```

Comment:
I don't think data-management tag is required here

Reply:
I've changed it already. Are there any more things to do? :)